src/components/etherpad-modal.jsx

```diff
@@ -0,0 +1,94 @@
+import {Component} from 'react';
+import * as BS from 'react-bootstrap';
+
+class Etherpad extends Component {
+  displayName: 'Etherpad'
```

Comment:
should be
```javascript
static displayName = 'Etherpad';
```

But anyway, not necessary since the name is the same as class name.

c.f. https://github.com/coala/gh-board/pull/17#discussion_r187311447